frontend/app/gameresult/page.tsx

```diff
@@ -119,7 +152,11 @@ const GameResult = () => {
                     }}
                   >
                     <Typography sx={{ fontSize: "2rem" }}>
-                      Win Rate: 70%
+                      Win Rate:{" "}
+                      {gameLog &&
+                        (gameLog?.user1win / (gameLog?.user1lose +
+                          gameLog?.user1win)) * 100}
+                      %
```

Comment:
gameLog.user1win gameLog.user1lose가 둘다 0 이면 nan 뜰거에요. 이에 대한 예외처리가 필요해보입니다!
```suggestion
                    <Typography sx={{ fontSize: "2rem" }}>
                      Win Rate:{" "}
                      {gameLog &&
                        (gameLog?.user1win / (gameLog?.user1lose +
                          gameLog?.user1win)) * 100}
                      %
```

Reply:
현재 적용 되었습니다.